src/lib/pages/Profile.svelte

```diff
@@ -36,6 +38,12 @@
       .collection(MongoCollections.Users);
     const result = (await data.find()) as UserData[];
     user = result.find((user) => user.personalId === realmApp.currentUser.id);
+    if (user != null) {
+      canSeePage = true;
+      if (user.accountState === 0) {
```

Comment:
bleh porovnavat enum s intem.... co by na to rekl tom :D

Reply:
Ten enum se tak ale zobrazuje a ukládá v tom mongu :D